### ISSUE TITLE
Add tests for every exposed metric

### ISSUE THREAD

**Issue by lidel** [ipfs/bifrost-gateway#31]
To avoid bugs like https://github.com/ipfs/go-libipfs/issues/162 we need to have regression test for every gateway metric fro go-libipfs.

**Comment by BigLep** [comment]
Yes, definitely agreed on needing to have our metrics documented.

One area I often see get missed is around latency metrics.  Please be clear on "when is the stopwatch started" and "when is the stopwatch ended".

**Comment by hacdias** [comment]
@lidel perhaps this tests would be better to live in `go-libipfs` instead, with the gateway code itself? I will investigate how to add tests for the metrics.
